src/patient-registration/validation/patient-registration-validation.tsx

```diff
@@ -1,8 +1,19 @@
 import * as Yup from 'yup';
 
 export const validationSchema = Yup.object({
-  givenName: Yup.string().required('Given name is required'),
-  familyName: Yup.string().required('Family name is required'),
+  givenName: Yup.string()
+    .required('Given name is required')
+    .test('alphabets', 'Only letters (A-Z,a-z) are allowed', value => {
```

Comment:
Sure that this is desired? I feel bad for `Ödin Manuél Enißán`.

Honestly, peoples names are complex. Sure, there will most likely not be a dollar sign in there, but its much more than 26 letters. See https://stackoverflow.com/questions/421046/what-are-all-of-the-allowable-characters-for-peoples-names (great links in there, e.g., https://www.w3.org/International/questions/qa-personal-names)